app/containers/SearchTrackContainer/tests/selectors.test.js

```diff
@@ -0,0 +1,25 @@
+// import { selectSearchTrackName } from '../selectors';
+
+describe('SearchTrackContainer selector tests', () => {
+  // let trackName;
+  // let mockedState;
+  // let searchedTrackName;
+  // let tracksData;
+
+  // beforeEach(() => {
+  //   trackName = 'sol';
+  //   tracksData = { items: [{ trackName }] };
+
+  //   mockedState = {
+  //     searchTrackContainer: {
+  //       searchedTrackName,
+  //       tracksData
+  //     }
+  //   };
+  // });
+
+  it('should select the trackName', () => {
+    // const trackSelector = selectSearchTrackName();
+    // expect(trackSelector(mockedState.searchTrackContainer)).toEqual(trackName);
+  });
+});
```

Comment:
remove if not required or write tests for them

Reply:
Yeah, I will write tests for them